02-cpu-configure/rffe-reset.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/sh
+
+set -eu
+if [ $# -ne 1 ]; then
+    echo "Usage: $0 <virtual_slot>"
+    exit 1
+fi
+
+expect << EOF
+spawn telnet 192.168.2.$(( 190 + $1 ))
+expect "NuttShell (NSH)"
```

Comment:
Shouldn't we wait for `nsh>` instead?

Reply:
I use [this](https://github.com/lnls-dig/bpm-cfg/blob/master/08-mmc-upload/fru_hard_rst.expect) as a reference, and in both cases it works by just looking for the first string printed in the shell after the connection is established.